repos/c2sm/packages/py-icon4py/package.py

```diff
@@ -27,8 +30,13 @@ class PyIcon4py(PythonPackage):
     depends_on('py-tabulate@0.8.9:', type=('build', 'run'))
     # TODO: push new version to Spack official
     depends_on('py-fprettify@0.3.7:', type=('build', 'run'))
-    depends_on('py-gt4py', type=('build', 'run'))
+    depends_on('py-gt4py@main:', type=('build', 'run'))
```

Comment:
I don't like that. `main:` has not really any meaning because it is not a number.
This also makes the life of the conretizer more difficult.

Either we stick to concrete version like `0.0.1:` or we don't specifiy it at all, and push that responsability to users or an environment file

Reply:
`main:` was a mistake. Removing the colon.

Reply:
I strongly recommend to hardcode the dependency to a branch that evolves on a daily base.